src/index.js

```diff
@@ -1,20 +1,35 @@
 import './style.css';
 import showMeals from './modules/showMeals';
+import { addLike } from './modules/fetchLikes';
+import showlike from './modules/showLike';
+import mealsCounter from './modules/mealsCounter';
 
 // Constants
 const mealsListContainer = document.querySelector('.f-list');
-const mealsNav = document.querySelector('.control');
 
 // Populate the meals cards items
-document.addEventListener('DOMContentLoaded', showMeals(mealsListContainer, 0));
+document.addEventListener('DOMContentLoaded', async () => {
+  await showMeals(mealsListContainer);
+  // Selecet meals items
+  const meals = document.getElementsByClassName('meal-card');
```

Comment:
## Well done, an excellent idea to calculate the items by their class name surely an idea I should include in future

Reply:
Thank you, I appreciate that 👏😎